client/state/jetpack-connect/actions.js

```diff
@@ -313,29 +313,22 @@ export function authorize( queryObject ) {
 				} );
 				// Update the user now that we are fully connected.
 				userFactory().fetch();
-				return wpcom.me().sites( {
-					site_visibility: 'all',
-					include_domain_only: true,
-					fields:
-						'ID,URL,name,capabilities,jetpack,visible,is_private,is_vip,icon,plan,jetpack_modules,single_user_site,is_multisite,options', //eslint-disable-line max-len
-					options:
-						'is_mapped_domain,unmapped_url,admin_url,is_redirect,is_automated_transfer,allowed_file_types,show_on_front,main_network_site,jetpack_version,software_version,default_post_format,created_at,frame_nonce,publicize_permanently_disabled,page_on_front,page_for_posts,advanced_seo_front_page_description,advanced_seo_title_formats,verification_services_codes,podcasting_archive,is_domain_only,default_sharing_status,default_likes_enabled,wordads,upgraded_filetypes_enabled,videopress_enabled,permalink_structure,gmt_offset,design_type', //eslint-disable-line max-len
```

Comment:
These fields appear to be missing compared with current behavior:

- `advanced_seo_front_page_description`
- `default_sharing_status`
- `design_type`
- `gmt_offset`
- `is_automated_transfer`
- `is_domain_only`
- `is_multi_network`
- `is_multi_site`
- `is_redirect`
- `main_network_site`
- `max_upload_size`
- `podcasting_archive`
- `publicize_permanently_disabled`
- `verification_services_codes`
- `videopress_enabled`
- `wordads`

Reply:
And these are included (weren't before):

- `ak_vp_bundle_enabled`
- `background_color`
- `blog_public`
- `default_category`
- `default_comment_status`
- `default_ping_status`
- `featured_images_enabled`
- `header_image`
- `headstart`
- `headstart_is_fresh`
- `image_default_link_type`
- `image_large_height`
- `image_large_width`
- `image_medium_height`
- `image_medium_width`
- `image_thumbnail_crop`
- `image_thumbnail_height`
- `image_thumbnail_width`
- `is_wpcom_store`
- `login_url`
- `post_formats`
- `site_goals`
- `theme_slug`
- `timezone`
- `woocommerce_is_active`